packages/react-components/react-nav-preview/src/components/NavCategory/useNavCategory.ts

```diff
@@ -12,14 +13,18 @@ import type { NavCategoryProps, NavCategoryState } from './NavCategory.types';
  * @param ref - reference to root HTMLDivElement of NavCategory
  */
 export const useNavCategory_unstable = (props: NavCategoryProps, ref: React.Ref<HTMLDivElement>): NavCategoryState => {
+  const { value } = props;
+
+  const { openItems } = useNavContext_unstable();
+
+  const open: boolean = openItems?.includes(value);
+
   return {
-    // TODO add appropriate props/defaults
+    open,
+    value,
```

Comment:
I don't see where the value is used, but for my previous comment about the callback, if this is used as an identifier, feel free to ignore.

Reply:
See previous comment :-)

Reply:
replied to the wrong comment - you're right, it's not used - deleting :-)

Reply:
Looks like children is used on the state object in renderNavCategory_unstable.